hangman.rb

```diff
@@ -0,0 +1,14 @@
+def hangman(word, letters)
+    result = ''
+    word.each_char do |i|
+        if letters.include?(word[i])
+            result += word[i]
```

Comment:
The block arguments for each_char are not integers but the characters of the string, which allows us to simplify the block:

```suggestion
    word.each_char do |c|
        if letters.include?(c)
            result += c
```